solutions/platinum/usaco-1188.mdx

```diff
@@ -0,0 +1,142 @@
+---
+id: usaco-1188
+source: USACO Platinum 2022 January
+title: Minimizing Haybales
+author: Aakash Gokhale
+---
+
+[External Solution (C++)](http://www.usaco.org/current/data/sol_prob1_platinum_jan22.html)
+
+## Explanation
+
+There are a lot of different ways to solve this problem. I am going to explain one that involves square root decomposition but I would recommend reading the official editorial to check out some of the other solutions as well.
```

Comment:
```suggestion
There's a ton of ways to solve this, but since you're probably here from the square root decomposition module, let's just go over that one.
The official editorial does have some nice alternative solutions as well.
```